Dockerfile

```diff
@@ -41,14 +41,17 @@ RUN export DEBIAN_FRONTEND=noninteractive \
     && apt-get purge -y --auto-remove $BUILD_DEPS \
     && apt-get autoremove -y \
     && apt-get clean \
-    && rm -rf /var/lib/apt/lists/* /root/.m2
+    && rm -rf /var/lib/apt/lists/* /root/.m2 \
+
+    # jolokia
+    && wget -q -O /opt/jolokia-jvm-1.3.7-agent.jar "http://search.maven.org/remotecontent?filepath=org/jolokia/jolokia-jvm/1.3.7/jolokia-jvm-1.3.7-agent.jar"
```

Comment:
Can we get this over https instead?

Reply:
Even better, would it be possible to use an ENV for the version number?

Reply:
@lmontrieux there's no way to get .jar over https. I updated it to use ENV.